prow/config.yaml

```diff
@@ -32,6 +32,9 @@ branch-protection:
           protect-by-default: true
         test-infra:
           protect-by-default: true
+    kubernetes-sigs:
```

Comment:
You'll need `protect-by-default: true` for this to actually do anything (otherwise it will just add `cla/linuxfoundation` to the ones that have this key

Reply:
forgot to hit save in my editor, thanks for the catch